src/Goteo/Model/LegalDocumentType/NIE.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Goteo\Model\LegalDocumenType;
```

Comment:
The namespace should be `LegalDocumentType`, not `LegalDocumenType`. Same applies to the rest of the classes in this namespace. All have the typo.